View/View.php

```diff
@@ -132,6 +133,21 @@ public function setData($data)
     }
 
     /**
+     * Set template variable
+     *
+     * @param $variableName
+     * @param $data
+     *
+     * @return View
+     */
+    public function setTemplateData($variableName, $data)
```

Comment:
@giosh94mhz you can also propose an alternative PR based on this PR .. might make the discussion easier.